cfe_internal/enterprise/federation/federation.cf

```diff
@@ -329,6 +329,9 @@ bundle agent clean_when_off
       "$(cfengine_enterprise_federation:config.path_setup_status)" -> { "ENT-7233" }
         comment => "We must remove this file for Mission Portal to understand that the federation is not configured",
         delete => default:tidy;
+      "$(cfengine_enterprise_federation:config.path_setup_status)" -> { "ENT-7233" }
```

Comment:
The promisee should be `ENT-7969`, right?

Reply:
yeah, right, changed